examples/introExample/build.gradle

```diff
@@ -10,13 +10,22 @@ android {
         versionCode 1
         versionName "1.0"
     }
-    buildTypes {
+
+    signingConfigs {
         release {
-            minifyEnabled false
-            //proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            storeFile file("keystore/release.keystore")
+            storePassword "realm1234"
```

Comment:
It is just a dummy certificate so the password is not secret. Ideally you would probably want to store it in an environment var or something, but doing it like this makes it work out of the box.